qa/rpc-tests/wallet_z_sendmany.py

```diff
@@ -63,148 +65,136 @@ def check_balance(self, node, account, address, expected, minconf=1):
 
     def run_test(self):
         # z_sendmany is expected to fail if tx size breaks limit
-        myzaddr = self.nodes[0].z_getnewaddress()
+        n0sapling = self.nodes[0].z_getnewaddress()
```

Comment:
The variable renames mixed in with the test fixes made this file a third of all the changes in the PR, and I really struggled to review it. Anything I missed is most likely to be in this file.

Reply:
I found it pretty much impossible to get it working without naming the variables absolutely consistently, so I'm loath to change this.